docs/usage.md

```diff
@@ -0,0 +1,240 @@
+]
+```
+
+In such a case, the notification email excludes the specified connector, its mentioned configuration and all versions. So, all versions of *Virus Total* and its configurations &mdash; `vt-1` and `vt-2` &mdash; are excluded.
+
+#### All Parameters Specified
+
+Consider the following snippet. In the exclude list, all the parameters are specified.
+
+```JSON
+[
+  {
+    "name": "VirusTotal",
+    "version": "",
+    "configName": [ "vt-1, "vt-2" ]
+  }
+]
+```
+
+Only the specified connector, with the specified version, and mentioned configuration is excluded.
+
+### Only Include List Specified
+
+This section explains the scenarios where certain parameters may be missing or unspecified in the include list.
+
+#### Connector `name` Specified
+
+Consider the following snippet. In the include list, only the name is provided.
+
+```JSON
+[
+  {
+    "name": "VirusTotal",
+    "version": "",
+    "configName": []
+  }
+]
+```
+
+In such a case, the notification email displays the health status of the specified connector(VirusTotal) with all its versions and configurations.
+
+>**WARNING**: If there is a typo in the connector's name or the connector is not installed, the connector appears in the **Invalid Include List** table as **Connector Not Present**.
+
+#### Connector `name` and `version` specified
+
+Consider the following snippet. In the include list, only the name and version is provided, but not the configuration name (`configName`).
+
+```JSON
+[
+  {
+    "name": "VirusTotal",
+    "version": "3.0.0",
+    "configName": []
+  }
+]
+```
+
+In such a case, the notification email displays the health status of the specified connector(VirusTotal), its mentioned version(3.0.0) and all its configurations.
+
+>**WARNING**: If the connector's version is incorrect or the specified version is not configured, the connector appears in the **Invalid Include List** table as **Connector Not Present**.
+
+#### Connector `name` and `configName` specified
+
+Consider the following snippet. In the include list, only the name and configuration name (`configName`) is provided, but not the version.
+
+```JSON
+[
+  {
+    "name": "VirusTotal",
+    "version": "",
+    "configName": ["vt-1", "vt-2" ]
+  }
+]
+```
+
+In such a case, the notification email displays the health status of the specified connector(VirusTotal), its mentioned configuration(vt-1 and vt-2) and all its versions.
+
+>**WARNING**: If the connector's version is incorrect, the connector appears in the **Invalid Include List** table as **Connector Not Present**.
+
+#### All Parameters Specified
+
+Consider the following snippet. In the include list, all the parameters are specified.
+
+```JSON
+[
+  {
+    "name": "VirusTotal",
+    "version": "3.0.0",
+    "configName": [ "vt-1, "vt-2" ]
+  }
+]
+```
+
+In such a case, the notification email displays the health status of the specified connector(VirusTotal), its specified version(3.0.0), and its mentioned configuration(vt-1 and vt-2).
```

Comment:
How does the health status notification will start? Does user will need to configure a schedule for it? If yes,How and it's frequency